scripts/acbuild-rkt-builder.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -ex
+
+MODIFY=${MODIFY:-""}
+FLAGS=${FLAGS:-""}
+IMG=${IMG:-"debian"}
+IMGVERSION=${VERSION:-"sid"}
+DOCKERIMG="$IMG:$IMGVERSION"
+ACIFILE=${ACIFILE:-"$IMG-$IMGVERSION.aci"}
+OUTACI="rkt-builder.aci"
+BUILDDIR=/opt/build-rkt
+SRC_DIR=/opt/rkt
```

Comment:
I'm changing them all to be snake case except `BUILDDIR` which is a variable used in the rkt build (im keeping that as-is to be consistent)